cmd/nginx-ingress/main.go

```diff
@@ -195,7 +196,7 @@ func main() {
 	}
 
 	if *versionFlag {
-		fmt.Printf("Version=%v GitCommit=%v\n", version, gitCommit)
+		fmt.Printf("Version=%v GitCommit=%v Date=%v\n", version, commit, date)
```

Comment:
is it clear from the context that it is a build date? Perhaps `BuildDate` ?

Reply:
I think it's clear from the context, like for the git commit, but I'm fine either way.